pkg/clusterlink/network/interface.go

```diff
@@ -42,3 +42,9 @@ func NewNetWork() NetWork {
 	dn.InitSys()
 	return dn
 }
+
+func NewAnalysisNetwork() NetWork {
+	dn := &DefaultNetWork{}
```

Comment:
recommend :
 implements : network interface Network

Reply:
ok, I've modified and add tag because for network analysis there is no need to perform 'initSys' logic again.